src/dojo/icon.m.css

```diff
@@ -1,7 +1,7 @@
 @import './variables.css';
 
 @font-face {
-  font-family: var(--icon-font-family);
```

Comment:
if this var is no longer used elsewhere can you please remove it?

Reply:
Well, it's still used when referring to the icon font in the other styles. i.e., the `.icon` class uses it and the listbox styles us it. Do you want me to hardcode the font family in all those places or just leave this?

Reply:
i think it should be hardcoded everywhere to be consistent

Reply:
Updated 👍